docsite/source/index.html.md

```diff
@@ -49,7 +49,10 @@ Currently, the railtie **does not make any assumptions about your directory/file
 ```ruby
 # config/initializers/system.rb
 Dry::Rails.container do
-  auto_register!("app/operations")
+  config.component_dirs.add "app/operations"
+  config.component_dirs.add "lib" do |dir|
+    dir.default_namespace = "my_super_cool_app"
+  end
```

Comment:
This is of course a breaking change, but afaik it was removed from dry-system as well
I guess it's better to keep the API the same instead of coming up with some magic used only in dry-rails 🤔